### PR TITLE
runtime: blanket application of namespacing and inclusion of `new`

### PR DESCRIPTION
**Pull request by compnerd**
Apply a blanket pass of including `new` for the placement new allocation
and namespacing the call to the global placement new allocator.  This
should repair the Android ARMv7 builds.

Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.

Resolves SR-NNNN.

### CODE REVIEW

**Comment by compnerd** [comment]
@swift-ci please test

**Review by grynspan — APPROVED** [review]
My only nitpick is that the system headers should be in alphabetical order, but I like a little chaos now and then.

Please run this by a member of the core team before merging.

**Review by mikeash — APPROVED** [review]

**Comment by compnerd** [comment]
@swift-ci please test Windows platform

**Comment by compnerd** [comment]
The windows failure is unrelated, going to merge it to repair the android build.

**Pull request merged**
